### PR TITLE
[WIP]API geolocalisation stored in articles

### PR DESCRIPTION
**Pull request by PaulineBA**
### User Story
```
As a user
In order to read relevant articles close to my location
I would like to see relevant articles to be based on my location.
```

### CODE REVIEW

File: spec/requests/api/admin/editor_can_add_location_spec.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+RSpec.describe 'Api::Admin::Articles :update', type: :request do
```
Comment (oliverochman) [inline spec/requests/api/admin/editor_can_add_location_spec.rb:1]: fix indentation in this file

**Review by oliverochman — CHANGES_REQUESTED** [review]
Update seed file to have location as well

**Review by erikbjoern — CHANGES_REQUESTED** [review]
After you've made sure the old tests pass, this looks good!

**Comment by erikbjoern** [comment]
This is being replaced with PR#22

**Pull request closed without merging**